answers/src/answers/Problem28.java

```diff
@@ -0,0 +1,31 @@
+package answers;
+
+/**
+ * Created by seok on 2017. 3. 6..
+ *
+ * Problem 28
+ *
+ * Starting with the number 1 and moving to the right
+ * in a clockwise direction a 5 by 5 spiral is formed as follows:
+ * (21) 22  23 24 (25)
+ * 20   (7)  8 (9) 10
+ * 19    6  (1) 2  11
+ * 18   (5)  4 (3) 12
+ * (17) 16  15 14 (13)
+ * It can be verified that the sum of the numbers on the diagonals is 101.
+ * What is the sum of the numbers on the diagonals
+ * in a 1001 by 1001 spiral formed in the same way?
+ */
+public class Problem28 {
+    public static void main(String[] args) {
+        System.out.println(spiralDiagonalsSum(501));
+    }
+
+    private static int spiralDiagonalsSum(int size) {
+        int sum = 1;
+        for (int i = 3; i < size * 2; i += 2) {
```

Comment:
Why `size * 2`, though `size` is never used elsewhere? Why not `spiralDiagonalsSum(1001)` and `i <= size`, which is what the problem is saying?